readme.md

```diff
@@ -219,6 +219,7 @@ If you want to contribute, please read the [contribution guidelines](contributin
 - [no-use-extend-native](https://github.com/dustinspecker/eslint-plugin-no-use-extend-native) - Prevent using extended native objects.
 - [Promise](https://github.com/xjamundx/eslint-plugin-promise) - Best practices when working with promises.
 - [pure](https://github.com/purely-functional/eslint-plugin-pure) - Enforce pure functions (without side effects).
+- [ReDoSDetector](https://github.com/tjenkinson/eslint-plugin-redos-detector) - ESLint plugin for finding possible ReDoS vulnerabilities.
```

Comment:
Note that I don't use it, here some discussion about the difference with `eslint-plugin-redos`: https://github.com/google/zx/pull/777#pullrequestreview-1999223033